scripts/earthly-ci

```diff
@@ -1,42 +1,49 @@
 #!/usr/bin/env bash
-# A wrapper for Earthly that is meant to caught signs of known intermittent failures and continue.
+# A wrapper for Earthly that is meant to catch signs of known intermittent failures and continue.
 # The silver lining is if Earthly does crash, the cache can pick up the build.
 set -eu -o pipefail
 
-# Flag to determine if -i is present
-INTERACTIVE=false
-# Check for -i flag in the arguments
```

Comment:
the interactive flag logic is removed as this is getting complex / doesnt fit well